deployments/local/README.md

```diff
@@ -1,9 +1,11 @@
-# Local Minio Deployment
+# Local Deployment
 
 ## Description
-This is an example local deployment of the Store Service that also starts up a local Minio server
-on port 9000, creates an ingest-quarantine bucket, and uses that for the S3 store. It also includes
-a deploy.sh script that will attempt to create the docker network needed.
+This is an example local deployment of the Store Service that also starts up a Localstack Docker image that provides
+an implementation of the S3 API on port 4572. It also creates 3 S3 buckets: ingest-quarantine, metacard-quarantine,
+and irm. A basic dashboard UI is available on port 8055 and a JSON record of all requests to S3 will be stored in
+<project-root>/.localstack/data/s3_api_calls.json .
```

Comment:
✏️ Please mention that the `.localstack` directory needs to be created beforehand